projects/lib/src/types.ts

```diff
@@ -1,14 +1,6 @@
 /**
- * Additional options that can be passt to tryLogin.
+ * Additional options that can be passed to tryLogin.
  */
-export class LoginOptions {
-  /**
-   * Is called, after a token has been received and
-   * successfully validated.
-   *
-   * Deprecated:  Use property ``events`` on OAuthService instead.
-   */
-  onTokenReceived?: (receivedTokens: ReceivedTokens) => void;
```

Comment:
This appears to be an accidental deletion. I checked out the branch and it doesn't build as this is missing.

Reply:
Yes, we already noticed, we also saw that it still uses refresh tokens with code flow. We adjusted this method but we don't know if this is the way to do it. If we add !this.silentRefreshRedirectUri it works with silent refresh.

`    
protected refreshInternal(params, noPrompt) {
        if (this.responseType === 'code' && !this.silentRefreshRedirectUri) {
            return this.refreshToken();
        } else {
            return this.silentRefresh(params, noPrompt);
        }
    }
`